builder/builder.Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:jessie
+
+RUN apt-get update && \
+    apt-get install -y curl debootstrap bzip2 && \
+    apt-get clean
+
+ENV WORKDIR /var/builder
+WORKDIR ${WORKDIR}
+
+ARG DOCKER_VERSION=1.11.2
+RUN curl -sSLO https://github.com/docker/docker/archive/v${DOCKER_VERSION}.tar.gz && tar -xzf v${DOCKER_VERSION}.tar.gz
```

Comment:
No, I don't need the docker binary, I only need the `contrib` directory in the `docker` repo so I can use `contrib/mkimage.sh`. The actual `docker build` of the final image happens outside of the builder image. I'll rename the builder image so it's less ambiguous about what it does.